styles/util/_layout.scss

```diff
@@ -1,7 +1,23 @@
 @use '../mixins/layout';
 
-// Apply vertical rhythm (margins between immediate-child elements) at the
-// default rhythm-unit size
+// Apply vertical rhythm (top/bottom margins between immediate-child elements)
+// at the default rhythm-unit size
 .u-vertical-rhythm {
   @include layout.vertical-rhythm;
 }
+
+// Apply horizontal rhythm (left/right margins between immediate-child elements)
+// at the default horizontal-rhythm size
+.u-horizontal-rhythm {
```

Comment:
Question, does this utility class allow the parameter in the `horizontal-rhythm`?  Like this `u-horizontal-rythm(1em)`, or it takes the default parameter.

Reply:
You're getting ahead of me! I was going to introduce additional utility classes as needed with modifiers indicating size. In the `client`, these look like, e.g., `.u-horizontal-rhythm--small`. We've been _sort of_ moving away from named unit sizes in `frontend-shared`, so I'm not sure how this will shake out precisely. But there will be one _class_ per "size".

Classes don't take parameters, but can wrap uses of mixins that provide different parameters. See the `.u-border--*` classes for examples.

I'll cross this bridge when we get closer to it...